ui/page/components/components.go

```diff
@@ -108,6 +108,31 @@ func UniformHorizontalPadding(gtx layout.Context, body layout.Widget) layout.Dim
 	}.Layout(gtx, body)
 }
 
+func UniformMobileHorizontalPadding(gtx layout.Context, withList bool, body layout.Widget) layout.Dimensions {
+	padding := values.MarginPadding10
+	if withList {
+		padding = 0
+	}
+	return layout.Inset{
+		Right: padding,
+		Left:  values.MarginPadding10,
+	}.Layout(gtx, body)
+}
+
+func UniformMobile(gtx layout.Context, withList bool, body layout.Widget) layout.Dimensions {
+
+	padding := values.MarginPadding10
+	if withList {
+		padding = 0
+	}
+
+	return layout.Inset{
+		Top:   values.MarginPadding24,
+		Right: padding,
+		Left:  values.MarginPadding10,
+	}.Layout(gtx, body)
+}
```

Comment:
The only difference i see with the 2 methods is the top inset. this can be merged into 1 and the inset passed as a parameter.

```suggestion
func UniformMobile(gtx layout.Context, isHorizontal, withList bool, body layout.Widget) layout.Dimensions {

	insetRight := values.MarginPadding10
	if withList {
		insetRight = values.MarginPadding0
	}
	
	insetTop := values.MarginPadding24
	if isHorizontal{
	     insetTop = values.MarginPadding0
	}
	
	return layout.Inset{
		Top:   insetTop,
		Right: insetRight,
		Left:  values.MarginPadding10,
	}.Layout(gtx, body)
}
```